SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1766,6 +1769,35 @@ void USpatialReceiver::OnCommandRequest(const Worker_Op& Op)
 				FSpatialTraceEventBuilder::CreateReceiveCommandRequest(TEXT("CLEAR_RPCS_ON_ENTITY_CREATION"), RequestId), Op.span_id, 1);
 		}
 
+		return;
+	}
+	else if (ComponentId == SpatialConstants::MIGRATION_DIAGNOSTIC_COMPONENT_ID
+			 && CommandIndex == SpatialConstants::MIGRATION_DIAGNOSTIC_COMMAND_ID)
+	{
+		check(NetDriver != nullptr);
+		check(NetDriver->Connection != nullptr);
+
+		if (EntityId == SpatialConstants::INVALID_ENTITY_ID)
+		{
+			UE_LOG(LogSpatialReceiver, Warning,
```

Comment:
This will log here, and for the invalid actor

Reply:
I think we can just remove this check entirely - if it's invalid it won't get an actor - which will error

Reply:
Sure, done